### PR TITLE
Initial packaging.

### LINKED ISSUE #1: Convert to namespace plugin and apply new template
Since @kdeldycke [graciously offered](https://github.com/getpelican/pelican-plugins/issues/1287) to help migrate the [Neighbors plugin](https://github.com/getpelican/pelican-plugins/tree/master/neighbors) from the legacy monolithic repository, I have created a new repository here and performed all the steps listed under **Initial Setup (By Maintainer)** here: https://blog.getpelican.com/drafts/namespace-plugin-migration.html

The remaining steps to be done are listed in the [**Updating the Plugin**](https://blog.getpelican.com/drafts/namespace-plugin-migration.html#updating-the-plugin) section further down the page. Be sure to change the example plugin name in the listed commands to match the current plugin name.

### PR DESCRIPTION
**Pull request by kdeldycke**
Package the bare copy of `neighbors` plugin into a independent python package, based on:

* https://github.com/getpelican/cookiecutter-pelican-plugin
* https://blog.getpelican.com/drafts/namespace-plugin-migration.html

Closes #1.

### CODE REVIEW

**Comment by kdeldycke** [comment]
@justinmayer: this PR is ready to be reviewed! 😃

File: readme.md [hunk 1]
```diff
@@ -60,9 +60,12 @@ template:
 ```
 
 
-## Usage with the Subcategory plugin
+## Subcategory plugin support
+
+This plugin is compatible with the [`subcategory`
+plugin](https://github.com/getpelican/pelican-plugins/tree/master/subcategory).
+here is how to use it.
```
Comment (justinmayer) [inline readme.md]: Maybe change this short sentence to:
> Following below are instructions on how to use Neighbors in conjunction with the Subcategory plugin.

**Comment by justinmayer** [comment]
I made one small suggestion for a sentence in the README. Otherwise, everything looks great. Well done! 🥇

File: readme.md (hunk 1 above)
Comment (kdeldycke) [reply readme.md]: Done in: https://github.com/pelican-plugins/neighbors/pull/2/commits/a1eceb2d12dae49ea0a9904a45e5c8bbab0b35ad

**Comment by justinmayer** [comment]
Merged to `main`. Thanks again for your work on this, Kevin!

**Comment by kdeldycke** [comment]
Thanks @justinmayer for your trust and your incledible fast merging! 👍

**Comment by kdeldycke** [comment]
Should we delete the `migrate` branch now?

And how about cutting a release to PyPi? I'm ready to test and fix it on my site right now.

**Comment by justinmayer** [comment]
Thanks for the reminder. Unneeded branch deleted.

Sure, we can add the CI/CD machinery now. It should be a quick step. I'll submit a new branch shortly...

**Comment by justinmayer** [comment]
@kdeldycke: PR #3 added in order to get PyPI auto-publication machinery in place. 🕺

**Pull request closed without merging**